### PR TITLE
Initial api

### PR DESCRIPTION
**Pull request by sentayhuJo**

### CODE REVIEW

File: backend/models/Location.js [hunk 1]
```diff
@@ -2,33 +2,33 @@ import { Model } from 'objection';
...
-            properties: {
-                location_id: { type: 'integer' },
-                adress_id: { type: ['integer', 'null'] },
-                lat: { type: 'string', minLength: 1, maxLength: 255 },
-                lon: { type: 'string', minLength: 1, maxLength: 255 },
-            },
-        };
+      properties: {
+        location_id: { type: 'integer' },
+        adress_id: { type: ['integer', 'null'] },
```
Comment (kkarimi) [inline backend/models/Location.js]: address

File: backend/models/Address.js [hunk 2]
```diff
@@ -3,46 +3,43 @@ import Branch from './Branch';
...
-                branch_id: { type: ['integer', 'null'] },
-                adress_line: { type: 'string', minLength: 1, maxLength: 255 },
-                city: { type: 'string', minLength: 1, maxLength: 255 },
-                postcode: { type: 'string', minLength: 1, maxLength: 255 },
-            },
-        };
+      properties: {
+        address_id: { type: 'integer' },
+        branch_id: { type: ['integer', 'null'] },
+        adress_line: { type: 'string', minLength: 1, maxLength: 255 },
```
Comment (kkarimi) [inline backend/models/Address.js]: address

Comment (sentayhuJo) [reply backend/models/Address.js]: pull makes every thing rollback...
Done

File: backend/models/Location.js (hunk 1 above)
Comment (sentayhuJo) [reply backend/models/Location.js]: done

**Pull request merged**